internal/capability/operator_install.go

```diff
@@ -56,9 +56,9 @@ func operatorInstall(ctx context.Context, opts ...auditOption) auditFn {
 				return err
 			}
 		}
-		options.Csv = *resultCSV
+		options.Csv = resultCSV
 		if resultCSV == nil {
-			options.Csv = operatorv1alpha1.ClusterServiceVersion{}
+			options.Csv = &operatorv1alpha1.ClusterServiceVersion{}
```

Comment:
What's more useful to us? A zero value for CSV? Or a nil? I feel like a nil would convey better information to the reporting functions.

Reply:
I agree with having a nil value for CSV for now.

Reply:
I think it cannot be nil, since it is required by the report. It errors out on the report when submitting a nil value.